web-model/src/main/java/com/oasis/webmodel/response/ResponseStatuses.java

```diff
@@ -0,0 +1,6 @@
+package com.oasis.webmodel.response;
+
+public class ResponseStatuses {
```

Comment:
no plural naming for class name
change it to ResponseStatus

Reply:
Commit ID: 279a1f3